docs/README.md

```diff
@@ -42,44 +54,99 @@ source ~/ChildProjectVenv/bin/activate
 pip install git+https://github.com/LAAC-LSCP/ChildRecordsData.git
 ```
 
-2. Install git-annex
+### 2. Install git-annex
 
 The package also requires git-annex in order to retrieve the datasets. It can be installed via `apt install git-annex` or `brew install git-annex`. It is encouraged to use a recent version of git-annex (8.2+).
 
-## Installing datasets
+## Getting some data
+
+You can either have some data of your own that you would like to use the package on, or you may know of some datasets that are already in this format that you'd like to reuse.
+
+It may be easier to start with an extant dataset. Here is the list that we know exists. Please note that the large majority of these data are NOT public, and thus if you cannot retrieve them, this means you need to get in touch with the data managers.
+
+### Extant datasets
+
+Instructions to download extant datasets can be found [here](http://laac-lscp.github.io/ChildRecordsData/REUSE.html).
```

Comment:
Missing so far but I guess this is intended

Reply:
yes, that's right -- PROJECTS relabeled to REUSE, so the html won't exist until this branch is merged with master.